tray.go

```diff
@@ -8,15 +8,17 @@ import (
 
 // Tray event names
 const (
-	EventNameTrayCmdCreate          = "tray.cmd.create"
-	EventNameTrayCmdDestroy         = "tray.cmd.destroy"
-	EventNameTrayCmdSetImage        = "tray.cmd.set.image"
-	EventNameTrayEventClicked       = "tray.event.clicked"
-	EventNameTrayEventCreated       = "tray.event.created"
-	EventNameTrayEventDestroyed     = "tray.event.destroyed"
-	EventNameTrayEventDoubleClicked = "tray.event.double.clicked"
-	EventNameTrayEventImageSet      = "tray.event.image.set"
-	EventNameTrayEventRightClicked  = "tray.event.right.clicked"
+	EventNameTrayCmdCreate                = "tray.cmd.create"
+	EventNameTrayCmdDestroy               = "tray.cmd.destroy"
+	EventNameTrayCmdSetImage              = "tray.cmd.set.image"
+	EventNameTrayCmdPopUpContextMenu      = "tray.cmd.popup.contextmenu"
```

Comment:
Could you replace this with:

`EventNameTrayCmdPopupContextMenu      = "tray.cmd.popup.context.menu"`

Reply:
Done.